src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator, Paint {
+    private Color color;
+    private String figureType;
```

Comment:
why do you need this. when you draw a figure, you can specify the type of figure there